letsencrypt/client.py

```diff
@@ -138,22 +138,22 @@ def obtain_certificate(self, domains, csr=None):
 
         # Save Certificate
         cert_path, chain_path = self.save_certificate(
-            certr, self.config.cert_path, self.config.chain_path)
+            certr, self.config.cert_dir, self.config.cert_dir)
```

Comment:
based on https://github.com/letsencrypt/lets-encrypt-preview/pull/439/files?diff=unified#r30873087 this change should be reverted as well, `save_certificate` takes a full path to a file rather than only a directory